source/developers/projects/engine/javascript-libraries.rst

```diff
@@ -0,0 +1,24 @@
+:is-up-to-date: True
+
+.. index:: JavaScript Libraries
+
+.. _javascript-libraries:
+
+====================
+JavaScript Libraries
+====================
+
+Crafter CMS has a collection of JavaScript libraries that enable easy interaction with several Crafter CMS modules/utilities for creating websites and applications.
+
+The libraries have the following npm package names:
+
+* `@craftercms/content <https://www.npmjs.com/package/@craftercms/content>`__ - Contains services for retrieving content and navigation using APIs offered by Crafter CMS.
+* `@craftercms/classes <https://www.npmjs.com/package/@craftercms/classes>`__ - Contains useful classes for developing craftercms websites & applications.
+* `@craftercms/models <https://www.npmjs.com/package/@craftercms/models>`__ - Contains data model definitions of different structures of Crafter CMS. This interfaces are useful when developing in TypeScript.
+* `@craftercms/redux <https://www.npmjs.com/package/@craftercms/redux>`__ - Contains tools for integrating your application with Crafter Engine and Crafter Search using Redux as the state container.
+* `@craftercms/search <https://www.npmjs.com/package/@craftercms/search>`__ - Contains tools for integrating your application with Crafter Search.
+* `@craftercms/utils <https://www.npmjs.com/package/@craftercms/utils>`__ - Contains various utilities useful when developing with Crafter CMS
+* `@craftercms/ice <https://www.npmjs.com/package/@craftercms/ice>`__ - Contains JavaScript utilities to use Crafter CMS In Context Editing in your Apps and Sites
+
+
+See the blueprint **Video Center** in the *Public Marketplace* when creating a site in Crafter Studio for an example of using the above JavaScript libraries.
```

Comment:
Shouldn't we just link this: https://marketplace.craftercms.org/blueprints/org.craftercms.blueprint.videoCenter